modules/update-upgrading-web.adoc

```diff
@@ -30,9 +30,12 @@ link:https://access.redhat.com/downloads/content/290[in the errata section] of t
 ====
 For production clusters, you must subscribe to a `stable-\*`, `eus-*` or `fast-*` channel.
 ====
-ifdef::openshift-origin[]
-such as `stable-4`.
-endif::openshift-origin[]
++
+[NOTE]
+====
+If you want to update your cluster to the next minor version, select a channel that corresponds to that minor version before beginning your update.
+If you cannot see an update path to the target minor version, keep updating your cluster to the latest patch release for your current version until the next minor version is available in the path.
+====
```

Comment:
@sheriff-rh whoops, forgot to replace this Note text with the same text I put in for modules/update-upgrading-cli.adoc above.

I'm assuming you'll be fine with that but just wanted to give you a heads up since it's a post-review change

Reply:
Looks great, thanks @skopacz1 !